homeassistant/components/updater/binary_sensor.py

```diff
@@ -18,15 +21,9 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
 class UpdaterBinary(CoordinatorEntity, BinarySensorEntity):
     """Representation of an updater binary sensor."""
 
-    @property
-    def name(self) -> str:
-        """Return the name of the binary sensor, if any."""
-        return "Updater"
-
-    @property
-    def unique_id(self) -> str:
-        """Return a unique ID."""
-        return "updater"
```

Comment:
Oh.. yeah I also changed these static properties to use `_attr_*` variants, forgot about that (did that commit a few hours ago)